src/bundle/Resources/config/routing.yml

```diff
@@ -418,6 +418,13 @@ ezplatform.version.remove:
     defaults:
         _controller: 'EzPlatformAdminUiBundle:Version:remove'
 
+ezplatform.version.has_conflict:
+    path: /version/has-conflict/{contentId}/{versionNo}/{languageCode}
+    options:
+    defaults:
+        _controller: 'EzPlatformAdminUiBundle:Version\VersionConflict:versionConflict'
```

Comment:
No quotes here

Reply:
As we agreed, quotes stay in place, to keep consistency in this file